drivers/CMakeLists.txt

```diff
@@ -13,3 +13,4 @@ add_subdirectory_ifdef(CONFIG_ADP536X adp536x)
 add_subdirectory_ifdef(CONFIG_ST25R3911B_LIB st25r3911b)
 add_subdirectory_ifdef(CONFIG_BT_LL_NRFXLIB bt_ll_nrfxlib)
 add_subdirectory_ifdef(CONFIG_NRF9160_GPS nrf9160_gps)
+add_subdirectory(flash_patch)
```

Comment:
I assume it is correct that this should always included.

However, is there any point in including it when no module which locks the flash is being used?

Reply:
I see now that the checks are all performed in the `flash_patch` cmakelist, so I guess you can disregard this comment.

Reply:
It's done this way because of the warning test.